app/controllers/events_controller.rb

```diff
@@ -39,6 +39,14 @@ def index
       format.html  { entries }
       format.csv   { render_csv(entries) }
       format.xlsx { render_xlsx(entries) }
+      format.ics { render_ical(entries) }
+    end
+  end
+
+  def show
+    respond_to do |format|
+      format.html  { entries }
+      format.ics { render_ical(entries) }
```

Comment:
There are no `entries` involved in the `show` action, just an `entry`. Refer to `CrudController` for the corresponding implementation.